benchmark/scanner/scanner.py

```diff
@@ -137,6 +137,7 @@ def check_line_from_meta(self, file_path: str, line_num: int) -> Tuple[LineStatu
                         self._increase_result_dict_cnt(row["Category"], "false_cnt")
                         return LineStatus.FALSE, project_id, file_id
         self.lost_cnt += 1
+        print(f"LOST: {file_path}:{line_num}", flush=True)
```

Comment:
It seems for debugging?
Would you like to add this line to PR?

Reply:
Yes. I think, the output should be added to point for missed markup.

Reply:
Okay let's leave this line.